### PR TITLE
Feat/pb-fq2bammeth

### PR DESCRIPTION
**Pull request by sateeshperi**

### CODE REVIEW

File: nextflow.config [hunk 1]
```diff
@@ -216,6 +217,12 @@ profiles {
         executor.cpus           = 4
         executor.memory         = 8.GB
     }
+    gpu {
+        use_gpu                 = true
+        docker.runOptions       = '-u $(id -u):$(id -g) --gpus device=0'
```
Comment (GallVp) [inline nextflow.config:222]: Why not `--gpus all`?

File: tests/gpu.config [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    // Base directory for nf-core/modules test data
+    modules_testdata_base_path   = 's3://ngi-igenomes/testdata/nf-core/modules/'
+    // Base directory for nf-core/methylseq test data
+    pipelines_testdata_base_path = 'https://github.com/nf-core/test-datasets/raw/methylseq/'
+
+    // Input data
+    input       = "${projectDir}/assets/samplesheet_gpu.csv"
+    fasta       = "${params.pipelines_testdata_base_path}/reference/genome.fa"
+    fasta_index = "${params.pipelines_testdata_base_path}/reference/genome.fa.fai"
+    outdir      = 'results'
```
Comment (GallVp) [inline tests/gpu.config:17]: `use_gpu = true`?

Without that, this is just a non-gpu test.

File: nextflow.config [hunk 3]
```diff
@@ -216,6 +217,12 @@ profiles {
         executor.cpus           = 4
         executor.memory         = 8.GB
     }
+    gpu {
```
Comment (GallVp) [inline nextflow.config:220]: If the pipeline is started with this profile, the specified settings will be applied to every process.

I think we should apply these settings only to `PARABRICKS_FQ2BAMMETH`. Or am I missing something?

File: tests/gpu.config (hunk 2 above)
Comment (sateeshperi) [reply tests/gpu.config:17]: the use_gpu param is set in the when.param block

**Comment by sateeshperi** [comment]
closing in favor of [bwameth standalone subworkflow](https://github.com/nf-core/modules/pull/7007)

**Pull request closed without merging**